data/assets/examples/tileprovider/tileproviderbylevel/bylevel.asset

```diff
@@ -0,0 +1,64 @@
+-- Basic
+-- This example file adds two layers to a globe. The first layer is used for further
+-- distances from the planet and when zooming into the planet, it gets replaced with the
+-- second layer. The place at which this transition occurs is determined by the "level",
+-- a number has to be between 2 and 22. Every increase in level means a transition point
+-- that is closer to the planet by a factor of two. Recommended reading for this example
+-- is the documentation on the DefaultTileProvider.
```

Comment:
Maybe we can skip over the duplicated parts from the description of the tile provider here? To me at least it makes it a bit confusing

Also maybe link to the default tile provider?